packages/editor/src/components/tabs/call/useCallData.ts

```diff
@@ -2,51 +2,49 @@ import { useDataContext } from '../../../context';
 import { CallData, Mapping } from '@axonivy/inscription-protocol';
 import produce from 'immer';
 import { useCallback } from 'react';
+import { Consumer } from '../../../types/lambda';
```

Comment:
cool interface: much easier to read for java guys :+1:

Reply:
That was the intention :smile: